content/blog/2022/07/psp-migration-script.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Let me help you migrate your PSPs to Kubewarden policies!
+authors:
+- José Guilherme Vanz
+date: 2022-07-07
+---
+
+As announced in past blog posts, Kubewarden have 100% coverage of the deprecated,
+and soon removed, Kuberernetes PSPs. If everything goes as expected the PSPs will
+be removed in Kubernetes v1.25. Which  is planned to 23rd August 2022.
+With our users in mind, the Kubewarden team wrote a script, which leverage the
+migration tool wrote by people from [AppVia](https://github.com/appvia/psp-migration),
+to migrate PSP automatically. With this script the PSP will be migrated for the
+equivalent Kuberwarden policies.
+
+The script is available in the [utils repository](https://github.com/kubewarden/utils/path/to/script)
+in the Kubewarden Github organization. It's quite simple. It will download the
```

Comment:
```suggestion
in the Kubewarden Github organization. It will download the
```

Reply:
I don't think this has been added in the latest commit or maybe I'm not able to see the latest changes?

Reply:
I may rebased the branch an lost this change. Sorry.